docs2/site/docs/migrations/migration8.md

```diff
@@ -246,6 +246,80 @@ At this time GraphQL.NET does not directly support the `MaxLength` and similar a
 implement your own attributes as shown above, or call the `Validate` method to set a validation
 function.
 
+### 6. Validation rules can read or validate field arguments and directive arguments
+
+Validation rules can now read or validate field arguments and directive arguments. This is useful
+for edge cases, such as when a complexity analyzer needs to read the value of a field argument
+to determine the complexity of the field. While it could also be used within custom validation rules
+to validate the value of field arguments or directive arguments, perhaps based on schema directives,
+it cannot be used to validate fields of input object graph types.
```

Comment:
Some parts of the text look like a formal math definition. I had to slow down and read it again. For example, this is too verbose
> Validation rules can now read or validate field arguments and directive arguments.

IMO, this is easier to read
> Validation rules can now read or validate field and directive arguments.

And this happens in many places along the text

```suggestion
Validation rules can now read or validate field and directive arguments. This is useful
for edge cases, such as when a complexity analyzer needs to read the value of a field argument
to determine the complexity of the field. While it could also be used within custom validation rules
to validate the value of field or directive arguments, perhaps based on schema directives,
it cannot be used to validate fields of input objects.
```